pkg/graveler/ref/resolve_ref_test.go

```diff
@@ -26,6 +26,7 @@ func TestResolveRef_Dereference(t *testing.T) {
 	var previous graveler.CommitID
 	for i := 0; i < 20; i++ {
 		c := graveler.Commit{
+			Version:      "commit:v1",
```

Comment:
The behaviour of `~` (and possibly `^`) matters a great deal with this change, so let's test it!

Reply:
1. Added test
2. Kept "commit:v1" as a marker - so no new identity will be created